webassembly/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM gitpod/workspace-full:latest
+
+USER gitpod
+
+RUN bash -cl "cargo install wasm-pack cargo-wasm cargo-generate \
+    && curl \"https://wasmtime.dev/install.sh\" -sSf | bash"
+
+RUN wget "https://github.com/CraneStation/wasi-sdk/releases/download/wasi-sdk-8/wasi-sdk_8.0_amd64.deb" \
+    && sudo dpkg -i wasi-sdk_8.0_amd64.deb
+
+
+RUN bash -cl "git clone \"https://github.com/emscripten-core/emsdk.git\" \
+    && cd emsdk \
+    && ./emsdk install latest \
+    && ./emsdk activate latest \
+    && source ./emsdk_env.sh"
+
+RUN bash -cl "echo -e \"\nsource /home/gitpod/emsdk/emsdk_env.sh\nclear\" >> ~/.bashrc"
+
+WORKDIR /opt
```

Comment:
I think it's better to make the `git clone` commands explicit about where they clone, e.g.:

```Dockerfile
RUN git clone https://... /tmp/location && \
    cd /tmp/location && \
    # ... && \
    rm -rf /tmp/location
```

as opposed to changing the `WORKDIR`, which might cause bad surprizes for downstream consumers of this image.

Reply:
Done!